libafl/src/observers/map/const_map.rs

```diff
@@ -72,6 +72,16 @@ impl<T, const N: usize> AsMut<Self> for ConstMapObserver<'_, T, N> {
     }
 }
 
+impl<T, const N: usize> SimpleHash for ConstMapObserver<'_, T, N>
+where
+    T: Hash,
+{
+    #[inline]
+    fn hash_simple(&self) -> u64 {
+        RandomState::with_seeds(0, 0, 0, 0).hash_one(self)
```

Comment:
This should use `hasher_std` from libafl_bolts

Reply:
Also, why not just `Hash`?

Reply:
After some playing around with it: `Hash` relies on std, wouldn't that be an issue?